src/ucp/am/eager_multi.c

```diff
@@ -37,7 +37,7 @@ ucp_am_eager_multi_bcopy_proto_probe(const ucp_proto_init_params_t *init_params)
                                UCP_PROTO_COMMON_INIT_FLAG_ERR_HANDLING |
                                UCP_PROTO_COMMON_INIT_FLAG_RESUME,
         .super.exclude_map   = 0,
-        .super.reg_mem_type  = UCS_MEMORY_TYPE_UNKNOWN,
+        .super.send_mem_type = UCS_MEMORY_TYPE_UNKNOWN,
```

Comment:
IMO for bcopy protocols, super.send_mem_type should be always host memory. which is ok because if SEND_ZCOPY is not set, we should not check for registration capability anyway. and it will be consistent with memory copy performance estimation.